k8s/kube-base/kustomization.yaml

```diff
@@ -23,10 +23,18 @@ configMapGenerator:
   - name: cm-httpd-conf
     files:
       - httpd.conf
+  - name: cm-my-conf
```

Comment:
I would put all three files in a single configmap, rather than splitting them up like this.

Reply:
I started doing it with 1 configmap, but it was easier to think about as 3 different configmaps.

Reply:
It's too bad you couldn't place you comment below, here, as it really deserves to be in this conversation as opposed to somewhere else.

It just makes this seem disjointed.

Ok, fine, I put all three files in a single confmap